backstop/scenarios/badge.js

```diff
@@ -0,0 +1,5 @@
+const { scenario } = require('../scenarioHelper');
+
+module.exports = [
+  scenario('basic', { selectors: ['#demo'], hideSelectors: ['h2'] }),
```

Comment:
Can we rename the scenario to "Type all" to better match the naming structure I'm trying to get passed in #155?

Reply:
Changed